Samples.Swift/Samples.Swift.xcodeproj/project.pbxproj

```diff
@@ -19,26 +19,11 @@
 		117AAAC12479FBCF00A63B34 /* CreateOrder.Request.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117AAAC02479FBCF00A63B34 /* CreateOrder.Request.swift */; };
 		117AAAC32479FBE100A63B34 /* AccessToken.Response.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117AAAC22479FBE100A63B34 /* AccessToken.Response.swift */; };
 		117AAAC52479FBF100A63B34 /* CreateOrder.Response.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117AAAC42479FBF100A63B34 /* CreateOrder.Response.swift */; };
-		117AAB0D247B066100A63B34 /* PayPalCheckout.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 117AAB07247B059700A63B34 /* PayPalCheckout.framework */; };
-		117AAB0E247B066100A63B34 /* PayPalCheckout.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 117AAB07247B059700A63B34 /* PayPalCheckout.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		11DF7A272481B1A7003B1AC2 /* CaptureOrder.Request.swift in Sources */ = {isa = PBXBuildFile; fileRef = 11DF7A262481B1A7003B1AC2 /* CaptureOrder.Request.swift */; };
 /* End PBXBuildFile section */
 
-/* Begin PBXCopyFilesBuildPhase section */
-		117AAB01247B029D00A63B34 /* Embed Frameworks */ = {
-			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
-			dstPath = "";
-			dstSubfolderSpec = 10;
-			files = (
-				117AAB0E247B066100A63B34 /* PayPalCheckout.framework in Embed Frameworks */,
-			);
-			name = "Embed Frameworks";
-			runOnlyForDeploymentPostprocessing = 0;
-		};
-/* End PBXCopyFilesBuildPhase section */
-
```

Comment:
Why remove this? It will cause a runtime dyld crash now if using Carthage

Reply:
Actually about this, if I don't remove this, I'll get this error (image attached) using Pods, then I'll have to manually remove it in XCode for Pods dependency to work.
![Screen Shot 2021-04-06 at 12 28 40 PM](https://user-images.githubusercontent.com/80476442/113745796-9fe84f80-96d3-11eb-8fbd-a8db21bc8d92.png)

I think I can either remove it, and specify in README Carthage section that users need to add Carthage framework to XCode.
Or keep it, and specify in README Cocoapods section that users need to remove Carthage framework in XCode.

I think the former might be better. What do you think? :smile: